File: AULA_02/JAVA/Triangulo.java
/* Triangulo.java
 * Dada as três dimensões de um triângulo
 * determinar se é realmente um triângulo e qual tipo ele é
 * Entrada: x,y,z (double)
 * Saída:  se é triângulo e qual tipo
 *
 * Autor: Fabrício Olivetti de França
 * Disciplina Processamento da Informação
 * Universidade Federal do ABC
 */



import java.util.Scanner;

class Triangulo{
	public static void main(String[] args) {
		double x, y, z; 
		Scanner leitor = new Scanner(System.in);

		// captura valores
		System.out.println("Forneça as 3 dimensões: ");
		x = leitor.nextDouble();
		y = leitor.nextDouble();
		z = leitor.nextDouble();

		if( x+y>z && x+z>y && y+z>x ){
			if( x==y && y==z ){
				System.out.println("Equilatero");
			}else if( x==y || y==z || x==z ){
				System.out.println("Isóceles");
			}else{
				System.out.println("Escaleno");
			}
		}else{
			System.out.println("Não é um triângulo");
		}

	}
}

File: AULA_02/Python/Triangulo.py
# -*- coding: cp1252 -*-
'''
Triangulo.py
 * Dada as três dimensões de um triângulo
 * determinar se é realmente um triângulo e qual tipo ele é
 * Entrada: x,y,z (double)
 * Saída:  se é triângulo e qual tipo

Autor: Fabrício Olivetti de França
Disciplina Processamento da Informação
Universidade Federal do ABC
'''


x = int( raw_input("x = ") )
y = int( raw_input("x = ") )
z = int( raw_input("x = ") )

if x+y > z and x+z > y and y+z > x:
    if x==y and y==z:
        print "Equilatero"
    elif x==y or y==z or x==z:
        print "Isóceles"
    else:
        print "Escaleno"
else:
    print "Não é um triângulo"
